dashboard/app/serializers/api/v1/pd/application_quick_view_serializer.rb

```diff
@@ -4,12 +4,4 @@ class Api::V1::Pd::ApplicationQuickViewSerializer < ActiveModel::Serializer
   def name
     "#{object.sanitize_form_data_hash[:first_name]} #{object.sanitize_form_data_hash[:last_name]}"
   end
```

Comment:
this too - can we do something the [Pd::Enrollment.full_name](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/enrollment.rb#L199-L203) property?